src/sonic-eventd/tests/eventd_ut.cpp

```diff
@@ -229,12 +223,12 @@ void run_pub(void *mock_pub, const string wr_source, internal_events_lst_t &lst)
 void debug_on()
 {
     /* compile with -D DEBUG_TEST or add "#define DEBUG_TEST" to include. */
-#ifdef DEBUG_TEST
+// #ifdef DEBUG_TEST
```

Comment:
Delete commented out code

Reply:
This entire function is removed as redundant.